recipe/build-nocgo-platform.bat

```diff
@@ -14,11 +14,6 @@ for %%F in (activate deactivate) do (
   if errorlevel 1 exit 1
 
   rem Copy the rendered [de]activate scripts to %PREFIX%\etc\conda\[de]activate.d.
-  copy "%%F-go-platform.bat" "%PREFIX%\etc\conda\%%F.d\%%F-%PKG_NAME%.bat"
+  copy "%%F-go-platform.bat" "%PREFIX%\etc\conda\%%F.d\%%F-z60-%PKG_NAME%.bat"
```

Comment:
I am using `z60` to make our activate script files be loaded after the gcc, fortran and g++ activation scripts.